tag_manager/data_layer_name.rst

```diff
@@ -0,0 +1,59 @@
+.. highlight:: js
+.. default-domain:: js
+
+Custom data layer name
+======================
+
+Introduction
+------------
+The data layer is a global JavaScript object, that can be used to pass information form the website to PPMS container. The default value for the data layer name is `dataLayer`.
+
+    .. versionadded:: 10.0
+        The name of the object is customizable. The purpose of this article is to describe the steps that need to be performed to set up custom data layer name.
```

Comment:
Ah, now I get it. So whole article describes how to use new feature (custom dataLayer names). In that case you were right. This `versionadded` should be at the start of the file. It will look like this:
![2019-09-03T143628](https://user-images.githubusercontent.com/4488774/64173760-40bbbc80-ce58-11e9-9932-f34f00af54ae.png)